scripts/setup-rg/rg-template.example.parameters.json

```
@@ -0,0 +1,30 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "projectPrefix": {
+            "value": "mlos-mysql-autotune"
```

```suggestion
            "value": "mlos-autotune"
```

Here's a cool thing about this all: we don't need to have separate storage/db/scheduler for different projects, so I don't think we should embed the name in the control pane portion of things.

The project name can be a part of a set of experiments, but basically each group that wants to run an mlos instance could share it across projects, or even across an org in order to pool benchmark data for wide spread analysis.
That's something GSL has talked about for instance, despite the fact that we may work on many different target projects.

Make sense?

---

Makes sense! Accidentally left `mysql` in a bunch of these names, but it is intended to be generic.